frontend/integration-tests/views/catalog-page.view.ts

```diff
@@ -1,6 +1,6 @@
-import { $, $$, browser, by, element, ExpectedConditions as until } from 'protractor';
+import { $, browser, by, element, ExpectedConditions as until } from 'protractor';
 
-export const catalogTiles = $$('.catalog-tile-pf');
+export const catalogTiles = $('.pf-l-gallery').$$('a.catalog-tile-pf');
```

Comment:
Why was this change needed?

Reply:
This seems to help, but only OLM scenarios which I gave up addressing at this time.